src/proc/rotation-filter.cpp

```diff
@@ -106,10 +132,42 @@ namespace librealsense {
             }
 
             case 2: {
-                rotate_frame< 2 >( static_cast< uint8_t * >( const_cast< void * >( tgt.get_data() ) ),
-                                   static_cast< const uint8_t * >( src.get_data() ) ,
+                if( format == RS2_FORMAT_YUYV )
+                {
+                    if( _value == 90 || _value == -90 )
+                    {
+                            LOG_ERROR( "Rotating YUYV format is disabled for 90 or -90 degrees" );
```

Comment:
Do we intend to support this at a future PR?

Reply:
No support for 90 and 270 degrees for YUYV format